apps/speed.c

```diff
@@ -2252,13 +2256,14 @@ int speed_main(int argc, char **argv)
     }
 
     if (doit[D_EVP_CMAC]) {
-        EVP_MAC *mac = EVP_MAC_fetch(NULL, "CMAC", NULL);
+        EVP_MAC *mac = EVP_MAC_fetch(app_get0_libctx(), "CMAC",
```

Comment:
This can leak the mac in subsequent gotos

Reply:
And it didn't before?????

Reply:
You just focused my attention on it :)

Reply:
You need blinkers :)

Reply:
I've pushed a fix.